### PR TITLE
Added 3 functions to prepare trajectories format

### PR DESCRIPTION
**Pull request by GeorgeEfstathiadis**
Added functions `remove_data`, `prepare_data` and `int2str` to prepare trajectories for return in readable format.

### CODE REVIEW

File: forest/bonsai/simulate_gps_data.py [hunk 1]
```diff
@@ -966,55 +966,89 @@ def gen_all_traj(person: Person, switches: Dict[str, int],
...
+    utc_start_stamp = datetime2stamp(utc_start, "UTC")
+
+    new = np.zeros((obs.shape[0], 6))
+    new[:, 0] = (obs[:, 0] + timestamp_s) * 1000
+    new[:, 1] = (obs[:, 0] + utc_start_stamp) * 1000
+    new[:, 2] = obs[:, 1]
+    new[:, 3] = obs[:, 2]
+    new[:, 4] = 0
+    new[:, 5] = 20
+    new = pd.DataFrame(
```
Comment (hackdna) [inline forest/bonsai/simulate_gps_data.py:1024]: Simply `return pd.DataFrame(...)`?

**Review by hackdna — CHANGES_REQUESTED** [review]

**Review by hackdna — CHANGES_REQUESTED** [review]
Thanks, almost there!

File: forest/bonsai/simulate_gps_data.py [hunk 2]
```diff
@@ -966,55 +966,89 @@ def gen_all_traj(person: Person, switches: Dict[str, int],
...
-    return secondwise
-
-def int2str(h):
-    if h<10:
-        return str(0)+str(h)
+
+def prepare_data(
+    obs: np.ndarray, timestamp_s: int, tz_str: str
+) -> pd.DataFrame:
+    """Perpares the data in a dataframe.
```
[the remark this answers is not in the document]
Comment (GeorgeEfstathiadis) [reply forest/bonsai/simulate_gps_data.py]: Oh I see, my bad

**Review by hackdna — APPROVED** [review]
I am assuming that you'll update `sim_GPS_data()` in a subsequent PR.

**Comment by GeorgeEfstathiadis** [comment]
Yes that will be the next PR

**Pull request merged**